<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Teleoperation</class>
 <widget class="QWidget" name="Teleoperation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-580</x>
     <y>-10</y>
     <width>1780</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#bf40b8;&quot;&gt;唯品会&lt;/span&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt; &lt;/span&gt;&lt;span style=&quot; font-size:18pt; color:#eeeeec;&quot;&gt;无人车配送后台监控系统&lt;/span&gt;&lt;span style=&quot; font-size:20pt; color:#eeeeec;&quot;&gt;   --&lt;/span&gt;&lt;span style=&quot; color:#eeeeec;&quot;&gt;遥操作&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_forward">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>642</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>前进</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_backward">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>679</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>后退</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_stop">
   <property name="geometry">
    <rect>
     <x>249</x>
     <y>679</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_turnRight">
   <property name="geometry">
    <rect>
     <x>249</x>
     <y>642</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>右转</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_start">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>679</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>启动</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_horn">
   <property name="geometry">
    <rect>
     <x>367</x>
     <y>679</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>鸣笛</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_light">
   <property name="geometry">
    <rect>
     <x>367</x>
     <y>642</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>开灯</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_turnLeft">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>642</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>左转</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_taskAsk">
   <property name="geometry">
    <rect>
     <x>521</x>
     <y>679</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>任务查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_taskSelect">
   <property name="geometry">
    <rect>
     <x>521</x>
     <y>642</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>任务选择</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_boxSelect">
   <property name="geometry">
    <rect>
     <x>688</x>
     <y>642</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>货柜选择</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_vehicleInfo">
   <property name="geometry">
    <rect>
     <x>688</x>
     <y>679</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>车辆信息</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
